azurerm/internal/services/network/validate/network_connection_monitor.go

```diff
@@ -0,0 +1,84 @@
+package validate
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network/parse"
+)
+
+func NetworkConnectionMonitorID(i interface{}, k string) (warnings []string, errors []error) {
+	v, ok := i.(string)
+	if !ok {
+		errors = append(errors, fmt.Errorf("expected type of %q to be string", k))
+		return
+	}
+
+	if _, err := parse.NetworkConnectionMonitorID(v); err != nil {
+		errors = append(errors, fmt.Errorf("Can not parse %q as a resource id: %v", k, err))
+		return
+	}
+
+	return warnings, errors
+}
+
+func NetworkConnectionMonitorHttpPath(v interface{}, k string) (warnings []string, errors []error) {
+	value := v.(string)
+
+	if !regexp.MustCompile(`^(/.*)?$`).MatchString(value) {
```

Comment:
Should we put more thoughts on the validation logic here? Otherwise, path like `//////` is also a valid input?

Reply:
updated